test/integration/api/version.bats

```diff
@@ -12,14 +12,14 @@ function teardown() {
 	start_docker 1
 	swarm_manage
 
-	# version
-	run docker_swarm version
+	# Get version output
+	out=$(docker_swarm version)
+
+	run bash -c "echo '$out' | grep swarm/"
```

Comment:
I think this test was originally overkill. Just run docker_swarm version and check that `$output =~ swarm/`